src/pulse3D/plate_recording.py

```diff
@@ -515,7 +516,8 @@ def to_dataframe(self, from_dataframe: bool = False) -> pd.DataFrame:
             time_steps = copy.deepcopy(first_well.force[0])
         else:
             max_time = max([w.force[0][-1] for w in self.wells if w])
-            time_steps = np.arange(interp_period, max_time + interp_period, interp_period)
+            min_time = min([w.force[0][0] for w in self.wells if w])
+            time_steps = np.arange(min_time, max_time + interp_period, interp_period)
```

Comment:
recordings start at 0.0 and `np.arange(interp_period, max_time + interp_period, interp_period)` was starting at 1 which is why it was all shifted one time point

Reply:
👍